CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
-## v0.1 [TBD]
+## v0.1 [2022-01-20]
 
 _What's new?_
 
-- TBD
+- Added 36 benchmarks and 142 controls to check Terraform AWS resources against security best practices for security best practices. Controls for following services have been added:
```

Comment:
Added 36 benchmarks and 142 controls to check Terraform AWS resources against security best practices. Controls for the following services have been added: